tests/unit/Listeners/CommentEventListenerTest.php

```diff
@@ -21,8 +21,9 @@
  *
  */
 
-namespace OCA\Deck\Activity;
+namespace OCA\Deck\Listeners;
```

Comment:
:grimacing: okay i guess you can do it that way

Reply:
I also addressed the review comment from Luka with it as the namespace of the actual class was changed, it made sense to adapt the text namespace as well ;)